src/jobs/push_build_to_bucket_job.yml

```diff
@@ -26,9 +26,13 @@ parameters:
     type: string
   source:
     type: string
+  build_env:
+    type: enum
+    enum: [ 'beta', 'prod']
+    default: beta
 steps:
-  - attach_workspace:
-      at: .
+  - js_restore_build_outputs:
+      build_env: << parameters.build_env >>
```

Comment:
does this command succeed if the environment specific directory does not exist on the workspace?
```
cp -a << parameters.build_env >>/. .
rm -rf << parameters.build_env >>
```

if not this here is a breaking change (because the unaware existing use of this job fails to restore the workspace from now on!